### ISSUE TITLE
Is there System.Text.Json support instead of Newtonsoft? 

### ISSUE THREAD

**Issue by juniormayhe** [confluentinc/confluent-kafka-dotnet#2380]
Description
===========

Is there System.Text.Json native support for .NET 8 projects? Wondering if we can avoid having Newtonsoft dependency brought by Confluent Kafka 

![image](https://github.com/user-attachments/assets/95cc9356-07f2-4ccb-980b-afc4ec1fe08c)

**Comment by rayokota** [comment]
Not currently

**Comment by Sonic198** [comment]
Hi, Is there any chance that this change will be made anytime soon? I event found some PR related to that.

https://github.com/confluentinc/confluent-kafka-dotnet/pull/2321/changes

**Comment by dominik-onerail** [comment]
Yes, it would be very helpful if we could use System.Text.Json instead of Newtonsoft.
